packages/cli/lib/commands/list.js

```diff
@@ -11,7 +11,8 @@ module.exports = async function () {
 		process.stdout.write('\n');
 		info('Available official templates: \n');
 
-		repos.map(repo => {
+		repos.forEach(repo => {
```

Comment:
Can we do `repos.filter(repo => !repo.archived)` instead to have this scale with us archiving more of them?

Reply:
Whoops, can indeed! Forgot that wasn't just a list of repo names.

Thanks